### ISSUE TITLE
Define "array" list column type, similar to dropdown form field

### ISSUE THREAD

**Issue by mittultechnobrave** [octobercms/october#2510]
Hello Guys,

I am using OctoberCMS and using one of its awesome plugins called as "Builder" http://octobercms.com/plugin/rainlab-builder and so far so good.

But i m stuck at one stage and confused how to achive this. This is my scenario below.

I have created one new field called as "status" in my one of the plugins in which i have key values like (0=>Inactive, 1=>Active) .. this works fine as i am able to insert or update values based on my status selection..

But the thing is, when i go to listing page, i have selected a "status" field but its showing me values 0 or 1 instead of Active or Inactive.

How can i show Active or Inactive instead of 0 and 1 using builder plugin or any other way.. I need to be able to search and sort as well ..

Can anyone guide me for the same ?

Thanks

**Comment by daftspunk** [comment]
We plan on adding a list column type that can handle this, similar to the way a form dropdown field works. Prototype code:

```
status:
    type: array
```

This will look up the attribute via an options method and grab the label from the current value.

**Comment by mittultechnobrave** [comment]
ok, eventually i found this solution http://stackoverflow.com/questions/40799476/show-dropdown-value-in-listing-search-instead-of-dropdown-key-using-builder-plug

**Comment by daftspunk** [comment]
Glad to hear it! That solution is a good alternative / workaround for sure.

**Comment by mittultechnobrave** [comment]
Thank you for help and support!!! Highly appreciated.

**Comment by Samuell1** [comment]
This has already PR here: https://github.com/octobercms/october/pull/4377

**Comment by daftspunk** [comment]
Implemented as an outcome of #4377

```
    status:
        label: Status
        type: selectable
```

**Comment by Eeesbksbk** [comment]
> We plan on adding a list column type that can handle this, similar to the way a form dropdown field works. Prototype code:
> 
> ```
> status:
>     type: array
> ```
> 
> This will look up the attribute via an options method and grab the label from the current value.

Why not add type array ????
